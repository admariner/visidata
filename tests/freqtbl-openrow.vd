sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample	OrderDate		freq-col		Shift+F	open Frequency Table grouped on current column, with aggregations of other columns
sample_OrderDate_freq	count		freq-col		Shift+F	open Frequency Table grouped on current column, with aggregations of other columns
sample_OrderDate_freq_count_freq		キ1	open-row		Enter	open current row with sheet-specific dive
sample_OrderDate_freq_1		キ2016-01-06	open-row		Enter	open current row with sheet-specific dive
