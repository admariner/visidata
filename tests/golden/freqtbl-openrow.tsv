OrderDate	Region	Rep	Item	Units	Unit_Cost	Total
2016-01-06	East	Jones	Pencil	95	1.99	189.05
